site/content/en/docs/Guides/metrics.md

```diff
@@ -207,24 +207,48 @@ kubectl port-forward deployments/grafana 3000 -n metrics
+
+You need to grant all the necessary permissions to the users (see [Access Control Guide](https://cloud.google.com/monitoring/access-control)). The predefined role Monitoring Metric Writer contains those permissions. Use the following command to assign the role to your default service account.
+
+```bash
+gcloud projects add-iam-policy-binding [PROJECT_ID] --member serviceAccount:[PROJECT_NUMBER]-compute@developer.gserviceaccount.com --role roles/monitoring.metricWriter
+```
 
 {{< alert title="Note" color="info">}}
-Cloud Operations for GKE (including stackdriver monitoring) is enabled by default on GKE clusters, however you can follow this [guide](https://cloud.google.com/stackdriver/docs/solutions/gke/installing#upgrade-instructions) if it is currently disabled in your GKE cluster.
+Cloud Operations for GKE (including Cloud Monitoring) is enabled by default on GKE clusters, however you can follow this [guide](https://cloud.google.com/stackdriver/docs/solutions/gke/installing#upgrade-instructions) if it is currently disabled in your GKE cluster.
 {{< /alert >}}
 
-The default metrics exporter installed with Agones is Prometheus. If you are using the [Helm installation]({{< ref "/docs/Installation/Install Agones/helm.md" >}}), you can install or upgrade Agones to use Stackdriver, using the following chart parameters:
+Before proceeding, ensure you have created a metrics node pool as mentioned in the Google Cloud [installation guide]({{< ref "/docs/Installation/Creating Cluster/gke.md" >}}).
+
+The default metrics exporter installed with Agones is Prometheus. If you are using the [Helm installation]({{< ref "/docs/Installation/Install Agones/helm.md" >}}), you can install or upgrade Agones to use Cloud Monitoring, using the following chart parameters:
 ```bash
 helm upgrade --install --wait --set agones.metrics.stackdriverEnabled=true --set agones.metrics.prometheusEnabled=false --set agones.metrics.prometheusServiceDiscovery=false my-release-name agones/agones --namespace=agones-system
 ```
 
-With this configuration only Stackdriver exporter would be used instead of Prometheus exporter.
+If you are using the [YAML installation]({{< ref "/docs/Installation/Install Agones/yaml.md" >}}), you will need to update the necessary parameters in the `install.yaml` file using the following commands:
 
-#### Using Stackdriver with Workload Identity
+```bash
+helm pull --untar https://agones.dev/chart/stable/agones-1.27.0.tgz && \
+cd agones && \
+helm template agones-manual --namespace agones-system  . \
+  --set agones.metrics.stackdriverEnabled=true \
+  --set agones.metrics.prometheusEnabled=false \
+  --set agones.metrics.prometheusServiceDiscovery=false \
+  > install-custom.yaml
+```
 
-If you would like to enable stackdriver in conjunction with [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity), there are a few extra steps you need to follow:
+```bash
+kubectl apply -f install-custom.yaml
+```
```

Comment:
🤔 I'm not sure about this section. I'm kinda figuring it's not necessary and possibly confusing.

If you are using the YAML install, running this command doesn't actually change the install.yaml that is hosted on github for the release - it just _gives you_ an yaml file. 

If someone is familiar with Helm and wants a yaml file instead, I would argue it's probably likely they already know how to do this.

If they aren't familiar with Helm, I'd worry this section is confusing. I'd be tempted to drop it.

Reply:
Hmm I definitely could have phrased that more accurately. Similarly to the Install Agones using YAML page, I do think having some guidance around what to do if you are using YAML would be helpful. An alternative would be to simply add a note linking to [this part](https://agones.dev/site/docs/installation/install-agones/yaml/#:~:text=Copy-,To,-change%20the%20configurable) of the page to change the parameters.

I'm also not opposed to dropping it but let me know what you think.

Reply:
Personally, I don't see a lot of people finding confusion with generating yaml from Helm -- but it's entirely possible that's a blind spot I have.

> An alternative would be to simply add a note linking to [this part](https://agones.dev/site/docs/installation/install-agones/yaml/#:~:text=Copy-,To,-change%20the%20configurable) of the page to change the parameters.

Seems like a good compromise to me 👍🏻  Could even do it as an "Note" box if you want to highlight it, e.g.

https://github.com/googleforgames/agones/blob/1cc9a3bc0afc8e7dc98810b1f6ec09be2aa59e72/site/content/en/docs/Guides/metrics.md?plain=1#L114-L116